public/js/chat.js

```diff
@@ -12,12 +12,14 @@
   let endbtn = document.querySelector('#endbtn');
```

Comment:
endbtn is also here.

Reply:
the one I used is the button which appers in confirm end chat dialog box.